### DISCUSSION TITLE
Vue is not working because of ESLint

### DISCUSSION THREAD

**Discussion by OnFireMohamed** [eslint/eslint#15525] [General]
Hello.
I have installed VueJS and I created a new VueJS project, when I want to run the project via "npm run serve" command it shows some errors.
![Untitled](https://user-images.githubusercontent.com/67835053/149637847-7c626c4e-0726-41c5-9289-17f5bd513776.png)

**Comment by aladdin-add** [comment]
duplicated: https://github.com/eslint/eslint/issues/15175
